pydarn/utils/constants.py

```diff
@@ -0,0 +1,22 @@
+# (C) Copyright SuperDARN Canada, University of Saskatchewan
+# Author: Marina Schmidt
+#
+# Disclaimer:
+# pyDARN is under the LGPL v3 license found in the root directory LICENSE.md
+# Everyone is permitted to copy and distribute verbatim copies of this license
+# document, but changing it is not allowed.
+#
+# This version of the GNU Lesser General Public License incorporates the terms
+# and conditions of version 3 of the GNU General Public License,
+# supplemented by the additional permissions listed below.
+#
+# Modifications:
+#
+
+# Earth's radius (km)
+Re = 6371
+EARTH_EQUATORIAL_RADIUS = 6378.137
```

Comment:
```suggestion
# Standard accepted IUGG global average of Earth's radius (km)
Re = 6371
# Earth radius at the equator (km)
EARTH_EQUATORIAL_RADIUS = 6378.137
```
I'm not fussy, but I know some people might be, so added more description to the two Earth radii to explain the difference.
And just in case anyone asks: International Union of Geodesy and Geophysics (IUGG) suggests the average global value used as Re should be 6371km.  (Ironically, the International Astronomical Union (IAU) suggests using the equatorial radius as Re, however, we're not measuring distances in Earth radii in pyDARN, so I'm on the IUGG side for this.)

I guess if we ever need it we can add the polar radius when applicable - couldn't remember if we used the polar radius as a semi-minor axis in rpos code.